app/scripts/views/sign_up.js

```diff
@@ -177,6 +177,14 @@ define(function (require, exports, module) {
         shouldFocusPassword: autofocusEl === 'password',
         showSyncSuggestion: this.isSyncSuggestionEnabled()
       };
+      if (this.getUserAgent().hasWebChannelSupport()) {
```

Comment:
More context, there isn't really anything I can think of that is different between fx_desktop_v3 and fx_fennec_v1 as far as the UI/messaging, but we can't make the assumption that will always be the case. Probably the most important current distinction is when we bucket metrics by context.